### DISCUSSION TITLE
Subtitle detection

### DISCUSSION THREAD

**Discussion by Vitocorl** [awawa-dev/HyperHDR#704] [Q&A] [answered]
I am using Hyperhdr, running from my PC, with wled as an ambilight system. I have a problem with subtitles being detected as a light source and the bottom TV leds light up white every time subs are displayed. 
Is there any workaround for this problem, or do i have to alter LED Hardware configuration, so that the bottom TV Leds do not work at all?

**Comment by awawa-dev** [comment, marked as answer]
Hi
in HyperHDR you have an LED designer editor thanks to which you can correct the position of each area that is averaged to the LED colors. If the subtitles appear at the bottom (but always with some margin to the bottom edge) and interfere with the displayed colors, you can shrink these areas so that the subtitles do not appear on them (may require increasing the capture resolution). However, it probably won't work if you use blackborder detection, because then the subtitles may appear in any area at the bottom with no margin and the only thing left to do is turn off individual LEDs in the LED designer editor.

**Comment by Vitocorl** [comment]
Thanks for the guidelines, i use blackborder detection, since most of the movies/series i watch use black borders, so i guess my only solution is to turn off the bottom middle LEDs...

**Comment by awawa-dev** [comment]
Yes, in this case you have to turn them off, but it shouldn't be difficult because you can do it with the mouse. You can also try to enable grouping for these LEDs or enlarge these areas, which will result in a smaller impact of the subtitles on the LEDs during averaging. BTW, it's probably a matter of personal preferences and the player's capabilities, but in my setup my player removes blackboarder so it stretches the image on the TV without blackbars: of course, it slightly disturbs the proportions, but contrary to what you might think, it's not noticeable or I've just gotten used to it.

**Comment by Vitocorl** [comment]
Thanks again, but i have tried removing the borders by stretching the visible area and i do not like the result at all. Please inform me in which section of the Hyperhdr configuration i can turn off individual LEDs. I was thinking of putting the bottom LED count number to 0, so that no LED of the bottom line works, but i would prefer it if i can turn off only some of them ...

**Comment by awawa-dev** [comment]
It's explained in the wiki: https://github.com/awawa-dev/HyperHDR/wiki/Quick-start#manual-led-layout-and-testing
